### PR TITLE
Create ActivityLog nuget package

### PR DESCRIPTION
**Pull request by TopSwagCode**

### CODE REVIEW

File: libraries/dotnet/EnergyOrigin.ActivityLog/EnergyOrigin.ActivityLog.Unit.Tests/EnergyOrigin.ActivityLog.Unit.Tests.csproj [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+    <PropertyGroup>
+        <TargetFramework>net8.0</TargetFramework>
+        <ImplicitUsings>enable</ImplicitUsings>
+        <Nullable>enable</Nullable>
+
+        <IsPackable>false</IsPackable>
+        <IsTestProject>true</IsTestProject>
+    </PropertyGroup>
+
```
Comment (tnickelsen) [inline libraries/dotnet/EnergyOrigin.ActivityLog/EnergyOrigin.ActivityLog.Unit.Tests/EnergyOrigin.ActivityLog.Unit.Tests.csproj:11]: Remember TreatWarningsAsErrors

File: libraries/dotnet/EnergyOrigin.ActivityLog/EnergyOrigin.ActivityLog.Unit.Tests/UnitTest1.cs [hunk 2]
```diff
@@ -0,0 +1,10 @@
+namespace EnergyOrigin.ActivityLog.Unit.Tests;
+
+public class UnitTest1
+{
+    [Fact]
+    public void Test1()
+    {
+        Assert.True(true);
```
Comment (tnickelsen) [inline libraries/dotnet/EnergyOrigin.ActivityLog/EnergyOrigin.ActivityLog.Unit.Tests/UnitTest1.cs:8]: Delete?

File: libraries/dotnet/EnergyOrigin.ActivityLog/EnergyOrigin.ActivityLog/EnergyOrigin.ActivityLog.csproj [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+        <Nullable>enable</Nullable>
+		<IsPackable>true</IsPackable>
+		<PackageId>EnergyOrigin.ActivityLog</PackageId>
+		<RepositoryUrl>https://github.com/Energinet-DataHub/energy-origin</RepositoryUrl>
+		<Authors>Energinet</Authors>
+		<Company>Energinet DataHub A/S</Company>
+		<PackageDescription>This library enables activity logging in Energy Origin.</PackageDescription>
+		<PackageLicenseExpression>Apache-2.0</PackageLicenseExpression>
+    </PropertyGroup>
+
```
Comment (tnickelsen) [inline libraries/dotnet/EnergyOrigin.ActivityLog/EnergyOrigin.ActivityLog/EnergyOrigin.ActivityLog.csproj:15]: TreatWarningsAsErrors

**Review by martinhenningjensen — APPROVED** [review]

File: libraries/dotnet/EnergyOrigin.ActivityLog/EnergyOrigin.ActivityLog.Unit.Tests/EnergyOrigin.ActivityLog.Unit.Tests.csproj (hunk 1 above)
Comment (TopSwagCode) [reply libraries/dotnet/EnergyOrigin.ActivityLog/EnergyOrigin.ActivityLog.Unit.Tests/EnergyOrigin.ActivityLog.Unit.Tests.csproj:11]: fixed

File: libraries/dotnet/EnergyOrigin.ActivityLog/EnergyOrigin.ActivityLog.Unit.Tests/UnitTest1.cs (hunk 2 above)
Comment (TopSwagCode) [reply libraries/dotnet/EnergyOrigin.ActivityLog/EnergyOrigin.ActivityLog.Unit.Tests/UnitTest1.cs:8]: Build errors without test project. Unit tests will be added in next part.

File: libraries/dotnet/EnergyOrigin.ActivityLog/EnergyOrigin.ActivityLog/EnergyOrigin.ActivityLog.csproj (hunk 3 above)
Comment (TopSwagCode) [reply libraries/dotnet/EnergyOrigin.ActivityLog/EnergyOrigin.ActivityLog/EnergyOrigin.ActivityLog.csproj:15]: fixed

**Review by tnickelsen — APPROVED** [review]

**Pull request merged**